supplementary_style_guide/glossary_terms_conventions/product_conventions/partner-and-hybrid-cloud-infrastructure.adoc

```diff
@@ -0,0 +1,762 @@
+
+[discrete]
+[[disconnected-deployment]]
+==== image:images/caution.png[with caution] disconnected deployment or installation (noun)
+*Description*: An installation or deployment method that has been modified so that access to the internet is not required, for example, for use in an airgapped environment.
+
+*Use it*: With caution - support levels vary, so the exact method and limitations must be explained clearly.
+
+*Incorrect forms*: offline deployment, airgapped deployment
+
+[discrete]
+[[physical-disk]]
+==== image:images/yes.png[yes] physical disk (noun)
+*Description*: A physical hardware component that stores data.
+
+*Use it*: Yes
+
+[discrete]
+[[virtual-disk]]
+==== image:images/yes.png[yes] virtual disk (noun)
+*Description*: A file or set of files that presents as a physical disk to a virtualized operating system.
+
+*Use it*: Yes
+
+[discrete]
+[[disk-encryption]]
+==== image:images/yes.png[yes] disk encryption (noun)
+*Description*: Generically, the encryption of data while it is stored on a disk. Also known as block device encryption or encryption at rest.
+
+*Use it*: Yes
+
+[discrete]
+[[dispersed-volume]]
+==== image:images/yes.png[yes] dispersed volume (noun)
+*Description*: A Gluster volume that uses erasure coding to write data across three or more bricks to ensure that data remains available even when a certain number of bricks is not available.
+
+*Use it*: Yes
+
+[discrete]
+[[distributed-file-system]]
+==== image:images/yes.png[yes] distributed file system (noun)
+*Description*: A file system that presents files from a number of different storage devices, potentially on many different machines and in many different locations, as a single interface to an end user or consuming service.
+
+*Use it*: Yes
+
+[discrete]
+[[distributed-volume]]
+==== image:images/yes.png[yes] distributed volume (noun)
+*Description*: A Gluster volume that distributes data across one or more bricks.
+
+*Use it*: Yes
+
+
+[discrete]
+[[distributed-dispersed-volume]]
+==== image:images/yes.png[yes] distributed-dispersed volume (noun)
+*Description*: A Gluster volume that distributes and erasure codes data across one or more bricks.
+
+*Use it*: Yes
+
+[discrete]
+[[distributed-replicated-volume]]
+==== image:images/yes.png[yes] distributed-dispersed volume (noun)
+*Description*: A Gluster volume that distributes and replicates data across multiple bricks.
+
+*Use it*: Yes
+
+[discrete]
+[[dynamically-provisioned-storage]]
+==== image:images/caution.png[with caution] dynamically provisioned storage (noun)
+*Description*: Generically, a repository of data that changes in size depending on the current demand of the application or the user.
+
+*Use it*: With caution - in OpenShift Container Platform and OpenShift Data Foundation, use "dynamic volume provisioning" to refer to the related StorageClass configuration parameter.
```

Comment:
Can you write "StorageClass" as "storage class" (that's how we write it in OpenShift

Also, any of these caveats should be listed above under the Description section. The "Use it" field should only show the values "yes" "no" or "with caution". Can you do a sweep to check, I think I saw a few other instances of this.